### DISCUSSION TITLE
Quick question: Updating runelite built from src after a game update?

### DISCUSSION THREAD

**Discussion by billdbrown** [runelite/runelite#17053] [Q&A]
I'm sure the answer is straightforward and I'm just missing something--osrs had an update pushed and my installed runelite updated no problem, but the one I have built from source won't. What do I need to do to get the built one using the latest game client?

Thanks!

**Comment by Nightfirecat** [comment]
I'm not entirely sure what you mean. Are you fetching the updated source after the release went out? If so, what are you seeing that isn't updated?
